src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -576,6 +576,21 @@
     )
     private Set<ClaimsCostFrom> divorceClaimFrom;
 
+    @CCD(
+        label = "Supporting documents",
+        hint = "Such as marriage/civil union certificate, proof of name changes, certified translations etc.",
+        typeOverride = TextArea,
+        access = {DefaultAccess.class}
+    )
+    private String supportingDocuments;
```

Comment:
How comes this field is of type `String` rather than being a Set of `SupportingDocuments` like the `cannotUploadDocuments` field?

Reply:
Is it because it is an enum?

Reply:
This will actually be metadata about the uploaded documents, e.g. id, url, name, upload time etc.

Reply:
ok that makes sense. Will this String field hold information/metadata about multiple documents? If so, how can we differentiate between the different documents?

Reply:
are you sure document metadata is represented as text(String) in ccd ? There is complex type for this not sure how this will look in CCD

Reply:
@tomxelliott there can be multiple documents uploaded, for now I was going to store this as a JSON array, but if there's a complex Java type for documents uploaded to https://github.com/hmcts/document-management-client then it might be better to use that. @nitinprabhu Do you have a link to the type or an example of it being used?

Reply:
@SebFlippence there is a Document type in CCD

Reply:
https://github.com/hmcts/nfdiv-case-api/blob/master/src/main/java/uk/gov/hmcts/divorce/document/model/DivorceDocument.java

I think you can use this but make sure you have all the fields